City-of-Helsinki/drupal-helfi-platform-config
# UHF-5892: News list paragraph backend changes

# News list paragraphin backend muutokset [UHF-5892](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5892)

## What was done
* Added "Title" and "Description" fields for the "News list" paragraph type
* Enabled the paragraph type in
  * "Page" content type
    * "Upper content region" and "Lower content region"
  * "TPR Service" entity type
    * "Lower content region"
  * "TPR Unit" entity type
    * "Lower content region"
* Displaying the news items in "Medium Teaser" view mode 

## How to install
* Set up KYMP instance
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-5892_news_list_paragraph_backend_changes`
* Run `make drush-updb drush-cr`

## How to test
* Check that the new fields exist in the paragraph type
* Check that the paragraph can be added to the mentioned content and entity types and content regions
* Check that the "Medium Teaser" view mode is added and in use on the "Helfi: News" external entity type
